packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -233,11 +234,11 @@ describe('backupNg', () => {
         status: 'failure',
         result: expect.any(Object),
       })
-
-      expect(noSuchObject.is(vmTask.result)).toBe(true)
     })
 
-    it('fails trying to run a backup job with a VM without disks', async () => {
+    // Not handled by the new backup implementation.
+    // See https://github.com/vatesfr/xen-orchestra/commit/0811da90146647aa2c0894d39b4bc41e39859453
+    it.skip('fails trying to run a backup job with a VM without disks', async () => {
```

Comment:
You can delete them as long as they are not used by the new backup implementation.

Reply:
What's the current behavior of all backup/replication methods for a VM without disks?

Reply:
The VM's export succeeds.

Reply:
- full backup works?
- full restoration works?
- delta backup works?
- delta restoration works?
- DR works?
- CR works?

Reply:
Yes all functionalities work.

Reply:
Then I guess we can simply remove this test as this special case don't exist anymore.

Reply:
Ah! the DR backup succeed but the resulted VM doesn't have the good name and tag.

Reply:
Please investigate and keep me posted.

Reply:
Any news on this?

Reply:
Fixed by 157c81b0e9f1b61b43c3ed1763097b3e7f19b427